modules/ROOT/pages/xml-sdk.adoc

```diff
@@ -1211,6 +1211,78 @@ To use the operations from the <<example2, example>> above in a Mule app, it is
 </mule>
 ----
 
+[[xml-sdk-tls]]
+== TLS Support
+
+Starting with Mule 4.6, XML SDK supports Transport Layer Security (TLS). Use TLS to enhance security for your app. XML SDK connectors can support the `<tls:context>` parameter in their configuration parameters.
+
+=== Parameter Declaration
+
+Use the `xmlns:tlsEnabled=true` annotation to indicate the component to support TLS. The annotated element does not necessarily have to be a top level element. In the following example, the annotated element is `<http:request-connection>` which is a child of the `<http:request-config>` top level element.
+
+[source,xml,linenums]
+----
+<http:request-config
+name="concur-httpreq-config" xmlns:connection="true"
+basePath ....>
+  <http:request-connection xmlns:tlsEnabled="true"
+  host=...>
+----
+
+The `<tls:context>` parameter can be specified only in one component. Using it on more than one component results in an error. 
+
+XML SDK adds a `<tls:context>` parameter in the configuration of the generated extension. This parameter is optional unless the `<tls:context>` parameter in the target element is required and doesn’t have a default value.
+
+When the extension is used, the supplied `<tls:context>` parameter is mapped to the config element with this annotation. 
```

Comment:
Active voice? Who/what uses the extension? (Will the audience know?)

(It's hard to know who's doing what here.)